docs/pages/includes/server-access/custom-installer.mdx

```diff
@@ -0,0 +1,84 @@
+{{ cloud="foo" matcher="bar" matchTypes="baz" }}
```

Comment:
@GavinFrazar I don't see the `cloud` var being used, why is it included?

Reply:
ah, so you're right. I can't remember what I was trying to do with that, but it's no longer needed.